docs/migration-guides/qiskit-backendv1-to-v2.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Migrate from BackendV1 to BackendV2
+description: How to update your code to use `BackendV2` instead of `BackendV1`.
+---
+
+# Migrate from `BackendV1` to `BackendV2`
+
+The Qiskit [`BackendV1`](/api/qiskit/1.4/qiskit.providers.BackendV1) class has been deprecated and will be removed from service. This migration guide describes the small adjustments you need to make if your are a user of a provider that upgraded from [`BackendV1`](/api/qiskit/1.4/qiskit.providers.BackendV1) to [`BackendV2`](/api/qiskit/qiskit.providers.BackendV2).
+
```

Comment:
I think it's worth mentioning that `qiskit_ibm_runtime` has always used `BackendV2` and `qiskit_aer` has also been using V2 since 0.13. So no actions are needed for users that use exclusively backends from those 2 packages.